examples/grpc-client/client.go

```diff
@@ -42,21 +42,7 @@ func newDexClient(hostAndPort, caPath, clientCrt, clientKey string) (api.DexClie
 	return api.NewDexClient(conn), nil
 }
 
-func main() {
-	caCrt := flag.String("ca-crt", "", "CA certificate")
-	clientCrt := flag.String("client-crt", "", "Client certificate")
-	clientKey := flag.String("client-key", "", "Client key")
-	flag.Parse()
-
-	if *clientCrt == "" || *caCrt == "" || *clientKey == "" {
-		log.Fatal("Please provide CA & client certificates and client key. Usage: ./client --ca-crt=<path ca.crt> --client-crt=<path client.crt> --client-key=<path client key>")
-	}
-
-	client, err := newDexClient("127.0.0.1:5557", *caCrt, *clientCrt, *clientKey)
-	if err != nil {
-		log.Fatalf("failed creating dex client: %v ", err)
-	}
-
+func testPassword(cli api.DexClient) error {
```

Comment:
"testPassword" seems like an odd name for this function. It makes it sound like a unit test. Maybe just "createPassword"?